### DISCUSSION TITLE
Left "effect" button not working

### DISCUSSION THREAD

**Discussion by JohnSmithCz** [GeekFunkLabs/fluidpatcher#67] [Q&A]
Hello, I built a squishbox (excellent thing BTW and many thanks to the author/s for his/her work), my hw is RPi 3B+, PCM5102, sw i think v0.7.12 and everything works and I hear beautiful sound. The right button selects sounds and also gets me to the menu, but the left button, when I press, it does nothing. Respectively, when I press it, it does the left function (like the right button when I turn it to the left), but no LED lights up and no effect at all.

I have the button connected to GND and GPI022. I have tested the button and LED with a multimeter and they are functional.

Do I have to activate the button or anyone have any idea what to do? Thank you kindly...

**Comment by albedozero** [comment]
Hi! The stompbutton sends MIDI messages that you can route to different things based on the bank/patch you have selected. I just updated the [documentation](https://github.com/albedozero/fluidpatcher/blob/master/programs.md#navigating-patchesmenus) to explain this. In menus, the stompbutton becomes a back/exit button, and tapping the rotary encoder goes forward, so you can use the SquishBox with your feet if needed (it's hard to rotate the encoder with shoes on :wink:) - so it sounds like your stompbutton is working, you just don't have it routed to anything. It sends momentary CC 30 and toggling CC 31 on channel 16 - I'll include some examples of routing the stompbutton soon. However, the LED should toggle on and off when you press the stompbutton. Could you have soldered it in backwards (i.e. long leg to ground)?
